src/Sentry.Protocol/User.cs

```diff
@@ -80,6 +91,12 @@ internal void CopyTo(User user)
             {
                 user.IpAddress = IpAddress;
             }
+
+            if (user.Extras == null)
```

Comment:
Here'd be `if (user.InternaOther== null)` because `Other will make sure there's an instance on the getter.